backend/src/bundles/users/types/user-gender.ts

```diff
@@ -0,0 +1,3 @@
+type Gender = 'male' | 'female';
+
+export { type Gender };
```

Comment:
Can it be better to make an enum?

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/98/commits/14fae07af2bc0621f3e3db570ea9f67bec727e50](url)